rustfmt.toml

```diff
@@ -1,3 +1,5 @@
+edition = "2021"
```

Comment:
It isn't needed (auto detected from Cargo.toml) but doesn't hurt to add

Reply:
This is needed as some code editing integration do not use cargo fmt but rustfmt directly.